homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -332,9 +344,7 @@ def parse_hue_api_put_light_body(request_json, entity):
             report_brightness = False
             result = True
 
-        elif (entity.domain == "script" or
-              entity.domain == "media_player" or
-              entity.domain == "fan"):
+        elif entity.domain in ["script", "media_player", "fan", "cover"]:
```

Comment:
import the consts from the respective components

Reply:
done